libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -2509,6 +2513,7 @@ def invoke(
                 self._invoke,
                 input,
                 self._config(config, self.func),
+                **kwargs,
```

Comment:
should we update the signature of `invoke` in Runnable? it'll probably catch a few other places where kwargs are missing?